Makefile

```diff
@@ -107,7 +109,14 @@ Dockerfile: Dockerfile.in
 	sed -e 's|@BASEIMAGE@|$(BASEIMAGE)|g' $< >$@
 
 test: vet fmt
-	GO111MODULE=on go test -mod vendor -timeout=1m -v -race $(BUILD_TAGS) ./...
+	GO111MODULE=on go test -mod vendor -timeout=1m -v -race -short $(BUILD_TAGS) ./...
```

Comment:
With `-short`, does it mean the e2e test is not actually run (since you skip it in `TestNPD`)?

Reply:
Correct. I plan to make the `test` Make rule only run unit tests, so that it can finish fast.

e2e test will be slower and more expensive. I don't expect every developer would run it for every small change they make.

e2e test will be used primarily in CI (and possibly pre-commit, we'll see). When developers do large refactoring work, they can choose to run e2e test manually.

Reply:
Oh, I see. This is for unit test.